www/content/tutorial.md

```diff
@@ -25,6 +25,10 @@
         <p>Roc doesn’t have a numbered release or an installer yet, but you can follow the install instructions for your OS<a href="/install/getting_started.html#installation"> here </a>. If you get stuck, friendly people will be happy to help if you open a topic in<a href="https://roc.zulipchat.com/#narrow/stream/231634-beginners"> #beginners </a>on<a href="https://roc.zulipchat.com/"> Roc Zulip Chat </a>and ask for assistance!</p>
     </section>
 
+## [LLM Docs](#llm-docs) {#llm-docs}
+
+We have experimental LLM-friendly text files for our [tutorial](/llms.txt) and [standard library](/builtins/llms.txt) that you can use to prompt your favorite LLM to answer your questions about Roc!
```

Comment:
So the tutorial content is copied in without any modification? Do we have plans to tailor this for LLMs in future? 

Also where is the builtins one coming from?

Reply:
Yes it's just copied without modification. I matched what I did in [this PR](https://github.com/roc-lang/roc/pull/7190) which adds a llms.txt to the document generation code. Anton wanted to keep it simple. That code adds the builtin/llms.txt endpoint. I happy to change it if you want something a little different.